From jacmet@gmail.com  Mon Sep 15 10:11:30 2008
From: Peter Korsgaard <jacmet@sunsite.dk>
Date: Mon,  8 Sep 2008 10:08:59 +0200
Subject: USB: fsl_usb2_udc: fix VDBG() format string
To: david-b@pacbell.net, gregkh@suse.de, leoli@freescale.com, linux-usb@vger.kernel.org
Cc: Peter Korsgaard <jacmet@sunsite.dk>
Message-ID: <1220861339-14967-1-git-send-email-jacmet@sunsite.dk>


Fixes:
drivers/usb/gadget/fsl_usb2_udc.c: In function 'dr_controller_setup':
drivers/usb/gadget/fsl_usb2_udc.c:229: warning: format '%p' expects type
'void *', but argument 3 has type 'int'

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/usb/gadget/fsl_usb2_udc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/gadget/fsl_usb2_udc.c
+++ b/drivers/usb/gadget/fsl_usb2_udc.c
@@ -223,7 +223,7 @@ static int dr_controller_setup(struct fs
 	fsl_writel(tmp, &dr_regs->endpointlistaddr);
 
 	VDBG("vir[qh_base] is %p phy[qh_base] is 0x%8x reg is 0x%8x",
-		(int)udc->ep_qh, (int)tmp,
+		udc->ep_qh, (int)tmp,
 		fsl_readl(&dr_regs->endpointlistaddr));
 
 	/* Config PHY interface */
